spec/models/grantee_spec.rb

```diff
@@ -16,16 +16,66 @@
   end
 
   # TODO: test for missing attribute
-  describe "#name" do
-    it "delegates to the attributes" do
-      expect(subject.name).to eq hses_grantee[:attributes][:name]
+  context "with all the attributes" do
+    subject { described_class.new(hses_grantee: hses_grantee) }
+    describe "#name" do
+      it "delegates to the attributes" do
+        expect(subject.name).to eq hses_grantee[:attributes][:name]
+      end
+    end
+
+    # TODO: test for missing attribute
```

Comment:
[suggestion] this TODO could be some `xit` pending tests so we're reminded to add them?